package.json

```diff
@@ -6,9 +6,11 @@
     "@testing-library/jest-dom": "^5.11.6",
     "@testing-library/react": "^11.2.2",
     "@testing-library/user-event": "^12.5.0",
+    "@types/node": "^18.11.18",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-scripts": "^4.0.1",
+    "typescript": "^4.9.4",
```

Comment:
I believe typescript and `@types/node` should be dev dependencies, as they wouldn't be present during run time.